### PR TITLE
[R4R] Restore supply endpoints

### PR DESCRIPTION
**Pull request by karzak**
Adds 
```
/vesting/circulatingsupply
/vesting/totalsupply
```

back to client (used by coin information sites like coin gecko).

### CODE REVIEW

**Comment by denalimarsh** [comment]
@karzak looks like there's some duplicated code from https://github.com/Kava-Labs/kava/pull/584, cleaning it up would make reviewing these changes easier.

File: x/validator-vesting/client/rest/query.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+
+	"github.com/kava-labs/kava/x/validator-vesting/types"
+)
+
+func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/vesting/circulatingsupply", getCirculatingSupplyHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/vesting/totalsupply", getTotalSupplyHandlerFn(cliCtx)).Methods("GET")
```
Comment (denalimarsh) [inline x/validator-vesting/client/rest/query.go:18]: An explicit route variable would be clearer, declare const "vesting". May avoid some confusion as the other modules all use `/{ModuleName}/path/to/endpoint`, this is the only one that doesn't use the module name.

File: x/validator-vesting/client/cli/query.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	"github.com/kava-labs/kava/x/validator-vesting/types"
+)
+
+// GetQueryCmd returns the cli query commands for the kavadist module
+func GetQueryCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	valVestingQueryCmd := &cobra.Command{
+		Use:   types.ModuleName,
```
Comment (denalimarsh) [inline x/validator-vesting/client/cli/query.go:18]: Can we update `types.ModuleName` to `vesting` so it matches the REST api? It looks like it won't conflict with "vesting" in the latest version of cosmos-sdk, but not 100% sure here.

Comment (karzak) [reply x/validator-vesting/client/cli/query.go:18]: That would be a breaking change, so can't update it as part of the client release.

Comment (denalimarsh) [reply x/validator-vesting/client/cli/query.go]: Ah, my comment wasn't clear - I meant adding a new route variable to use instead of `types.ModuleName`

Comment (denalimarsh) [reply x/validator-vesting/client/cli/query.go]: resolved by https://github.com/Kava-Labs/kava/pull/586/commits/12a904214881a7ddf80ebd465c7f49674aaba68e

File: x/validator-vesting/client/rest/query.go (hunk 1 above)
Comment (denalimarsh) [reply x/validator-vesting/client/rest/query.go]: resolved by https://github.com/Kava-Labs/kava/pull/586/commits/12a904214881a7ddf80ebd465c7f49674aaba68e

**Pull request merged**